test/e2e/service.go

```diff
@@ -246,6 +249,94 @@ var _ = Describe("Services", func() {
 		}()
 	}, 240.0)
 
+	It("should be able to create a functioning external load balancer", func() {
```

Comment:
There's a lot of duplicated code between this test and some of the others in this file.  Suggest factoring out common code (in a separate PR, which I'll probably tackle today :-) ).